### PR TITLE
[masterbots.ai] feat: update ChatChatbotDetails

### PR DESCRIPTION
**Pull request by Bran18**
https://github.com/user-attachments/assets/ce012940-4bf8-4e5c-b60d-d6f64c267675

Screenshot 2024-11-25 at 12 36 00 PM

### Note: BotMessageSquare Error

The BotMessageSquare from https://lucide.dev/icons/bot-message-square its not working. I am using MessageSquare instead in the meantime 

## Summary by Sourcery

Update the ChatChatbotDetails component with a new UI layout, including a card header, floating avatar, and enhanced description section. Refactor the component for improved readability and maintainability.

New Features:
- Introduce a new UI layout for the ChatChatbotDetails component, including a card header, floating avatar, and enhanced description section.

Enhancements:
- Refactor the ChatChatbotDetails component to improve readability and maintainability by reorganizing the JSX structure and updating class names for styling consistency.

## Summary by CodeRabbit

- **New Features**
	- Improved user interface with a welcoming message in the card header.
	- Enhanced responsiveness and visual appeal of the component.
	- Added descriptive text to assist users in creating new threads.
	- Styled button for starting a new chat for better visual integration.
	- Introduced a new loading skeleton for the chatbot details interface.

- **Documentation**
	- Expanded documentation detailing features like responsive design and theme support.

- **Refactor**
	- Restructured the return statement for improved layout and styling.
	- Updated import paths for better modularity and organization.

### CODE REVIEW

File: apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx [hunk 1]
```diff
@@ -1,107 +1,149 @@
...
+                >
+                  Follow
+                </Button>
+              </div>
             </div>
+
+            <Button
+              className={cn(
+                'w-auto px-4 md:px-6 py-2 text-sm md:text-base',
+                'bg-[#be16e8] hover:bg-[#be16e8]/90',
```
[the remark this answers is not in the document]
Comment (Bran18) [reply apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx:149]: These green and purple colors are not the bg-primary. Our primary is almost grey.  Im not following this suggestion

File: apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx [hunk 2]
```diff
@@ -1,107 +1,162 @@
...
               </div>
+
+              <div className="flex items-center gap-2 md:gap-3">
+                <div className="flex items-center gap-1">
+                  <Users className="w-4 h-4 text-zinc-950 dark:text-gray-300" />
+                  <span className="text-xs font-normal leading-tight md:text-sm text-zinc-950 dark:text-gray-300">
+                    Followers:{' '}
+                    <span className="text-zinc-500 dark:text-zinc-500">
+                      3.2k
+                    </span>
```
[the remark this answers is not in the document]
Comment (Bran18) [reply apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx:132]: Yes, its ok. It's just a temporary value. Real data will eventually came from the DB

**Review by AndlerRL — APPROVED** [review]
LGTM! Thanks for dealing with the skeletons and update their file names. Also for the initial onboarding variant, it is looking good in desktop! Only the mobile need to be updated, but also the designs wasn't 100% ready at that moment. Merging this version!

**Comment by AndlerRL** [comment]
[Fix Browse Sidebar & Page Navigation](https://trello.com/c/54RpZyon/272-fix-browse-sidebar-page-navigation)

**Pull request merged**